sessionctx/variable/sysvar.go

```diff
@@ -594,6 +602,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, "character_set_connection", mysql.DefaultCharset},
 	{ScopeGlobal, MyISAMUseMmap, "0"},
 	{ScopeGlobal | ScopeSession, "ndb_join_pushdown", ""},
+	{ScopeGlobal | ScopeSession, "character_set_server", mysql.DefaultCharset},
```

Comment:
Why adding `binlog_order_commits`, `master_verify_checksum` and `character_set_server`?

Reply:
A mistake introduced by merging with the master, I will remove them.